docs/guide/usage/debug.md

```diff
@@ -30,15 +30,11 @@ To enable debug logging in the Zigbee2MQTT Docker container add `-e DEBUG=zigbee
   `zigbee_herdsman_debug: true`
 - Click `Save`, and when prompted to restart, click `Restart add-on`
 
-Herdsman debug logs should now sow up on the `Logs` tab for the addon.
+Herdsman debug logs should now show up on the `Logs` tab for the addon. 
 
-To grab all the logs, log in via SSH and execute:
+Logs are also stored in timestamped folders under `/config/zigbee2mqtt/log` directory. 
 
-```
-docker logs CONTAINER_ID > log.txt 2>&1
```

Comment:
This should be kept as this is the only way to grab the herdsman debug logs (they don't end up in the log files)